WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -27,20 +30,26 @@ protocol PointOfSaleAggregateModelProtocol {
 class PointOfSaleAggregateModel: ObservableObject, PointOfSaleAggregateModelProtocol {
     @Published private(set) var orderStage: PointOfSaleOrderStage = .building
 
+    @Published private(set) var cardReaderConnectionStatus: CardPresentPaymentReaderConnectionStatus = .disconnected
```

Comment:
At least based on these changes, card connection looks like it could be its own thing. `PointOfSaleConnectionModel` could also be reached from the environment by those who need it. Something to think about in the future. Maybe we will find domains that are not intertwined and could be kept separate.